modules/get-started/pages/architecture.adoc

```diff
@@ -40,7 +40,7 @@ https://raft.github.io/[Raft^] uses a heartbeat mechanism to maintain leader aut
 
 As long as there is a timing inequality between heartbeat time, election timeout, and mean time between node failures (MTBF), then Raft can elect and maintain a steady leader and make progress. A leader can maintain its position as long as one of the ten heartbeat messages it sends to all of its followers every 1.5 seconds is received; otherwise, a new leader is elected.
 
-If a follower triggers an election, but the incumbent leader subsequently springs back to life and starts sending data again, then it's too late. As part of the election process, the follower (now an election candidate) incremented the term and rejects requests from the previous term, essentially forcing a leadership change. If a cluster is experiencing wider network infrastructure problems that result in latencies above the heartbeat timeout, then back-to-back election rounds can be triggered. During this period, unstable Raft groups may not be able to form a quorum. This results in partitions rejecting writes, but no data loss can occur. Redpanda has a Raft-priority implementation that allows the system to settle quickly after network outages.
+If a follower triggers an election, but the incumbent leader subsequently springs back to life and starts sending data again, then it's too late. As part of the election process, the follower (now an election candidate) incremented the term and rejects requests from the previous term, essentially forcing a leadership change. If a cluster is experiencing wider network infrastructure problems that result in latencies above the heartbeat timeout, then back-to-back election rounds can be triggered. During this period, unstable Raft groups may not be able to form a quorum. This results in partitions rejecting writes, but previously written data will not be lost. Redpanda has a Raft-priority implementation that allows the system to settle quickly after network outages.
```

Comment:
very minor tweak to the wording, otherwise +1 on everything. 

> but data previously written to disk will not be lost

this reinforces the fact that as long as we've gotten it down to disk, we should be safe.